temporal-testing-junit4/src/main/java/io/temporal/testing/TestWorkflowRule.java

```diff
@@ -354,4 +354,27 @@ public History getWorkflowExecutionHistory(WorkflowExecution execution) {
   public Worker getWorker() {
     return testEnvironment.getWorkerFactory().getWorker(getTaskQueue());
   }
+
+  /**
+   * Returns the default test watcher impl that can be used is your tests. It simplifies the test watcher
+   * creation to just:
+   * <pre><code>
+   *   @Rule
+   *   public TestWatcher watchman = testWorkflowRule.getTestWatcher();
+   *   </code></pre>
+   *
+   * @return default test watcher impl
+   */
+  public TestWatcher getTestWatcher() {
+    return
+      new TestWatcher() {
+        @Override
+        protected void failed(Throwable e, Description description) {
+          if (getTestEnvironment() != null) {
+            System.err.println(getTestEnvironment().getDiagnostics());
```

Comment:
Please use logger, there are almost 0 valid cases when non-trivial app should be performing input and output directly to System.err or System.out.

Reply:
This is ran in tests and from what i see logger statements are swallowed. There is already a System.err in this class on like 87 ;)